api/serializers/errors.py

```diff
@@ -0,0 +1,13 @@
+from django.db.models import Model
+from rest_framework import serializers
+
+
+class ValidationErrorSerializer(serializers.Serializer[Model]):
+    """
+    Used only in OpenAPI schema, unless we are going to redefine DRF default errors serializer.
+    """
+
+    # This means that we can have multiple errors for any field of serializer.
+    field1 = serializers.ListField(child=serializers.CharField())
+    field2 = serializers.ListField(child=serializers.CharField())
```

Comment:
I wonder why there are exactly two fields

Reply:
It could be any amount of them, but it's kind of common format to show this in schema

Reply:
I'll have to think about this a bit more, OK for now :)

Reply:
You could check how it looks like for "jsonfield" (which could have any fields inside) in schema by default :)

Reply:
Well, I get the `serializers.ListField(child=serializers.CharField())` bit, but I don't understand why you have `field1` and `field2`, exactly two fields. Sorry :)

Reply:
Could you offer any other option to show that there are could be any amount of fields? For me these two "unnamed" fields show it fine :)

Reply:
Example which I mean is how JsonField example is rendered by spectacular by default 
<img width="302" alt="image" src="https://user-images.githubusercontent.com/38218668/227750868-7ed4f8ec-053c-46a0-84c4-d529bab2def9.png">

Reply:
But what if there are more than 2 fields? Is the developer supposed to subclass the `ValidationErrorSerializer`? Two fields `field1` and `field2` are like a magic number to me.

How does AutoSchema know what to do with these `field...` attributes of the serializer?

Damn, I don't like `drf-spectacular` docs at all. No meaningful docstrings even for classes. Makes it very hard to understand some parts. That's why maybe I completely misunderstand the issue. 

(BTW we could alias these `serializers` in import so that it's immediately clear that this is not our serializer, but that's just a thought)

Reply:
I think we should discuss it by voice, because I think you don't understand in general why I wrote this serializer  :)

Reply:
Yep, the plan is to make me understand this, then tweak the docstring so that any potential reader understands this :)

Reply:
I guess my question is why you decided not do use the "standard" approach mentioned in [the docs](https://www.django-rest-framework.org/api-guide/exceptions/#custom-exception-handling).